src/storage/v2/storage.cpp

```diff
@@ -1251,6 +1267,9 @@ utils::BasicResult<StorageIndexDefinitionError, void> Storage::DropIndex(
   commit_log_->MarkFinished(commit_timestamp);
   last_commit_timestamp_ = commit_timestamp;
 
+  // We don't care if there is a replication error because on main node the change will go through
+  memgraph::metrics::DecrementCounter(memgraph::metrics::LabelIndexCreated);
```

Comment:
If we increment and decrement counters, shouldn't the name be: ActiveLabelIndices and ActiveLabelPropetyIndices, not Created?

Reply:
Same for other metrics, not sure if it occurs at other places.

Reply:
Yes, I don't have actually counters for active indexes, just created, like the streams. But I can quickly create it as this were the legacy counters

Reply:
oh i actually have them, but the name is misleading